src/amo/components/LanguageTools/index.js

```diff
@@ -15,7 +15,7 @@ import {
   ADDON_TYPE_LANG,
   VIEW_CONTEXT_LANGUAGE_TOOLS,
 } from 'core/constants';
-import languages from 'core/languages';
+import { languagesUnfiltered as languages } from 'core/languages';
```

Comment:
I would use `languages` here, as before.

Reply:
Can you expand on what you're thinking?

Reply:
I'd require `languages` as before and only create an array of `sortedLanguages`. I dont see why you need this a named export in one file only to alias it to its original name here.

Reply:
See my other answer which relates to the same thing.